src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteUploadCommand.java

```diff
@@ -189,6 +190,12 @@ private Challenge getChallenge(final String checkName,
         final String challengeName = String.join(" - ", this.getCountryDisplayName(countryCode),
                 result.getName().isEmpty() ? checkName : result.getName());
         result.setName(challengeName);
+        // Add challenge name to check-in comment
+        result.setCheckinComment(String.format("#maproulette: %s", challengeName));
```

Comment:
Do we want the `#` to be configurable? As in referring to a different tag

Reply:
add a switch for this @seancoulter

Reply:
added*